src/routes/demos/save/.Part3.5MeetingTypesOptionsSIPFields.svelte

```diff
@@ -32,93 +110,117 @@
       </button>
     {/if}
   </div>
-  <div class="column pt-0 is-third">
+  <div class="column pt-0 is-one-quarter">
     <label class="label" for="extention-number"
       >Extension Number<sup class="has-text-danger" title="required">*</sup></label
     >
     <div class="control has-icons-left">
       <input
```

Comment:
Adding a regex check on the number may help avoid invalid input from the user.

Reply:
will create a JIRA for webex Node stuff